projects/espace/api/compose.yml

```
@@ -0,0 +1,69 @@
+      # You may use a bind-mounted host directory instead, so that it is harder to accidentally remove the volume and lose all your data!
+      # - ./docker/db/data:/var/lib/postgresql/data:rw
+###< doctrine/doctrine-bundle ###
+
+volumes:
+  caddy_config:
+  caddy_data:
+  database_data:
+  mailpit_data:
+
+###> doctrine/doctrine-bundle ###
+  database_data:
```

YAMLlint (1.37.1)

[error] duplication of key "database_data" in mapping

(key-duplicates)